### ISSUE TITLE
Not usable with Amazon Fire TV/Stick remote

### ISSUE THREAD

**Issue by Tarazet** [twireapp/Twire#121]
Thanks for the great work i like the app and use it already on my Smartphone.

Now i thought this would be great on my Amazon Fire Stick to avoid ads from Twitch.
It turns out i can't use it since i can't control the app / get past the starting screen with the Amazon Fire Stick remote.
I would like the control to be enhanced to work with input from fire tv remote.

**Comment by samfundev** [comment]
Someone else brought this up but they closed their issue: #117. Not sure why but I'll try looking into this. I don't have an Amazon Fire but I believe it can be emulated.

**Comment by Tarazet** [comment]
you basically have a remote to controll this and no touch function. I think it's comparable with using a keyboard with arrow keys, enter and the back key. i can imagine if the app can be controlley by a remote keyboard with arrow keys it can be controlled with the amazon remote, too.
But that's just all a guess from using Amazon Fire Stick on my end.

**Comment by mkrupczak3** [comment]
Bump @twireapp

I think the previous commenter is wrong, I believe the FireTV remote functions more like a Bluetooth gamepad than a keyboard just from my experiments running Freedoom on my Fire stick (that app actually worked, mind you)

**Comment by mkrupczak3** [comment]
I may fork this app and try to get it compliant with Android TV stuff if I have time. Sad state of the world I have to do this i guess

**Comment by Perflyst** [comment]
You are welcome to submit a PR ;)

**Comment by mkrupczak3** [comment]
Yeah I don't envy you, The Android ecosystem is a hot mess 🥵. Diving back in should be interesting
